eiffel-vocabulary/EiffelServiceStartedEvent.md

```diff
@@ -0,0 +1,21 @@
+
+# EiffelSeriveStartedEvent (ArtS)
+
+The EiffelServiceStartedEvent declares that a service has been started and is ready for use.
```

Comment:
I think this could be elaborated, as it isn't obvious what the relationship is between ServiceStarted and ArtifactDeployed.